ingest/Snakefile

```diff
@@ -60,7 +60,7 @@ if SEGMENTED:
         for segment in config["nucleotide_sequences"]:
             dataset_server_map[segment] = segment_identification.get(
                 "nextclade_dataset_server_map", {}
-            ).get(segment, segment_identification.get("nextclade_dataset_server"))
+            ).get(segment, segment_identification.get("nextclade_dataset_server", "https://data.clades.nextstrain.org/v3"))
```

Comment:
I'm hesitant to setting defaults so deeply in code.

We could just set this as the default in the config - then we don't need to add it everywhere we use the server? Less chance for divergence/bugs?

Reply:
ok! added to the defaults - its a bit ugly though as the server should only be set if segment_identification is set